LICENSE copy.md

```diff
@@ -0,0 +1,9 @@
+All Reports in this Repository are licensed by Contributors
+under the
+[W3C Software and Document License](http://www.w3.org/Consortium/Legal/2015/copyright-software-and-document).
+
+Contributions to Specifications are made under the
+[W3C CLA](https://www.w3.org/community/about/agreements/cla/).
+
+Contributions to Test Suites are made under the
+[W3C 3-clause BSD License](https://www.w3.org/Consortium/Legal/2008/03-bsd-license.html)
```

Comment:
I am not sure whether those to items apply, do they?
I TD [LICENSE.md](https://github.com/w3c/wot-thing-description/blob/main/LICENSE.md) it is not there.

Reply:
Not sure why TD does not have this information but my guess would that its license is simply older than what W3C asks for now